development/secrets-rotator/terraform/environments/dev/main.tf

```diff
@@ -60,3 +60,26 @@ module "service_account_keys_rotator" {
 output "service_account_keys_rotator" {
   value = module.service_account_keys_rotator
 }
+
+module "service_account_keys_cleaner" {
+  source = "../../modules/service-account-keys-cleaner"
+
+  application_name = var.application_name
+  service_name     = var.service_account_keys_cleaner_service_name
+  project = {
+    id     = data.google_project.project.project_id
+    number = data.google_project.project.number
+  }
+  region                                     = var.region
+  service_account_keys_cleaner_account_id    = var.service_account_keys_cleaner_account_id
+  service_account_keys_cleaner_image         = var.service_account_keys_cleaner_image
+  cloud_run_service_listen_port              = var.cloud_run_service_listen_port
+  scheduler_name                             = var.service_account_keys_cleaner_service_name
+  secrets_rotator_sa_email                   = google_service_account.secrets-rotator.email
+  scheduler_cron_schedule                    = var.service_account_keys_cleaner_scheduler_cron_schedule
+  service_account_key_latest_version_min_age = var.service_account_key_latest_version_min_age
```

Comment:
It's more of a nitpick, but is it really required to add the full name of the tf chart? The variables start to be a little lengthy... Especially since you define module name in L64, so I don't see a need to repeat the same name for every variable that you define for it.

Reply:
But it doesn't make sense to repeat that in every variable name under a module. You already know module name and repeating that name in every variable is not nice for reviewing